frontend/.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-build
\ No newline at end of file
+build
+.eslintrc.js
```

Comment:
please remove this one and can you please include this one in `tsconfig.json` as discussed

Reply:
this will solve the issue